src/components/SlateEditor/plugins/details/index.tsx

```diff
@@ -154,15 +160,15 @@ export const detailsPlugin = (editor: Editor) => {
     const { attributes, children, text } = props;
     const path = ReactEditor.findPath(editor, text);
 
-    const [parent] = Editor.node(editor, Path.parent(path));
+    const [parent] = Editor.node(editor, Path.parent(Path.parent(path)));
```

Comment:
Går du her to nivå opp?

Reply:
Jepp, Node elementet vil kun gi meg `{text: "..."}` så første nivå over vil være paragraph så kommer summary